vouch/vouch.yml

```diff
@@ -0,0 +1,40 @@
+# This is a sample config to run vouch VC. It is used by the run.sh script
+# to generate a custom config for each vouch VC connected to a charon node.
+# Refer: https://github.com/attestantio/vouch/blob/master/docs/configuration.md.
+
+# Address of the charon node. Ex: http://node0:3600
+beacon-node-address:
+
+# The wallet account manager obtains account information from local wallets, and signs locally.
+# It supports wallets created by ethdo.
+accountmanager:
+  wallet:
+    locations: /opt/vouch/keys
+    accounts: validators
+    passphrases: T8BFYJZU5R
+
+# metrics is the module that logs metrics, in this case using prometheus. Note that vouch doesn't emit metrics if
+# the following block is not provided.
+metrics:
```

Comment:
add this to `prometheus.yml` if not present there

Reply:
done